lib/constructor.js

```diff
@@ -145,6 +145,8 @@ const Sharp = function (input, options) {
     pngCompressionLevel: 6,
     pngAdaptiveFiltering: true,
     webpQuality: 80,
+    webpAlphaQuality: 100,
+    webpLossless: false,
     tiffQuality: 80,
```

Comment:
Is this missing a `webpNearLossless: false` default? This might be the reason for the "Partial image extraction WebP" test failure.